File: Semester 4/Programming and Projection Enironments/mpp-proiect-java/Proiect_MPP/AirportPersistence/src/main/java/root/persistence/people/employees/IEmployeeRepository.java
package root.persistence.people.employees;

import root.model.people.Employee;
import root.persistence.people.IPersonRepository;

import java.util.Collection;

public interface IEmployeeRepository extends IPersonRepository<Employee> {
    Employee findByEmail(String email);
    Collection<Employee> getByPosition(String position);
}

File: Semester 4/Programming and Projection Enironments/mpp-proiect-csharp/Proiect_MPP/persistence/people/employees/IEmployeeRepository.cs
using model.people;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace persistence.people.employees
{
    public interface IEmployeeRepository : IPersonRepository<Employee>
    {
        Employee? findByEmail(string email);
        List<Employee> getByPosition(string position);
    }
}
